### PR TITLE
v1.1.0 - Pass an instance of `PausableTimer` in the callback.

### PR DESCRIPTION
**Pull request by vxern**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
+  instead.
+- Pass an instance of the `PausableTimer` in the callback.
+
 ## 1.0.0+5
 
 - Update and test against Dart 2.17.
 
-[More details available in
-GitHub](https://github.com/llucax/pausable_timer/milestone/9?closed=1).
+[More details available in GitHub](https://github.com/llucax/pausable_timer/milestone/9?closed=1).
```
Comment (llucax) [inline CHANGELOG.md:11]: Could you remove all the reformatting changes from the changelog?

**Review by llucax — CHANGES_REQUESTED** [review]
Hi @vxern, thanks for opening the PR!

As I said in #48, I'm not convinced it is a good idea to add the instance to the callback.

Can you please split the changes into different PRs? I would like to merge the removal of the `clock` dependency (as a point release) but the other change is more controversial.

File: CHANGELOG.md (hunk 1 above)
Comment (vxern) [reply CHANGELOG.md:11]: > Could you remove all the reformatting changes from the changelog?

What is wrong with them? The file was automatically formatted by the Dart formatter when I saved it.

**Comment by vxern** [comment]
> Hi @vxern, thanks for opening the PR!
> 
> As I said in #48, I'm not convinced it is a good idea to add the instance to the callback.
> 
> Can you please split the changes into different PRs? I would like to merge the removal of the `clock` dependency (as a point release) but the other change is more controversial.

Sure. Closing.

File: CHANGELOG.md (hunk 1 above)
Comment (llucax) [reply CHANGELOG.md:11]: That is markdown, not dart, so maybe some other tool did them? There is nothing wrong with them, but there is nothing wrong with how it is formatted either, so I don't see any reason to change it and add noise to the diff. It is distracting and maybe the next person that opens a PR thinks there is a better formatting too (or has another tool to automatically format it). It just makes no sense to change it, unless it is a standard tool that is making the auto formatting and we can add a check to the CI to make sure the same format is always used (as it is with the dart code).

**Pull request closed without merging**